plugins/outputs/wavefront/sample.conf

```
@@ -51,6 +50,9 @@
   ## Telegraf.
   #immediate_flush = true
 
+  ## Send internal metrics (starting with `~sdk.go`) for valid, invalid, and dropped metrics. default is true.
+  # send_internal_metrics = false
```

If the default is true, then the following line should also say true
```suggestion
  ## Send internal metrics (starting with `~sdk.go`) for valid, invalid, and dropped metrics. default is true.
  # send_internal_metrics = true
```